tasks/options/ngtemplates.js

```diff
@@ -78,21 +78,10 @@ module.exports = {
                     return 'export default [];\r\n';
                 }
 
-                // abs returns the correct absolute path to be used with a require call. On Windows,
```

Comment:
What about the Windows problem?

Reply:
these should be gone because it doesn't put there abs path but module name and let webpack to resolve it